include/libopencm3/stm32/fdcan.h

```diff
@@ -461,17 +414,14 @@ LGPL License Terms @ref lgpl_license
 
 /* TFFL[2:0]: Tx FIFO free level */
 #define FDCAN_TXFQS_TFFL_SHIFT			0
-#define FDCAN_TXFQS_TFFL_MASK			0x7
 
 /* TFGI[1:0]: Tx FIFO get index */
-#define FDCAN_TXFQS_TFGI_SHIFT			0
-#define FDCAN_TXFQS_TFGI_MASK			0x3
+#define FDCAN_TXFQS_TFGI_SHIFT			8
 
 /* TFQPI[1:0]: Tx FIFO put index */
-#define FDCAN_TXFQS_TFQPI_SHIFT			0
-#define FDCAN_TXFQS_TFQPI_MASK			0x3
+#define FDCAN_TXFQS_TFQPI_SHIFT			16
 
-#define FDCAN_TXFQS_TFQF				(1 << 0)
+#define FDCAN_TXFQS_TFQF				(1 << 21)
```

Comment:
this looks like it was a standalone bug fix?

Reply:
Yes, sorta. Somehow it never broke functionality of the old G4 code. Is it absolutely necessary to keep it in separate pull request?